.docker/nginx-datahub-mock.conf

```diff
@@ -0,0 +1,40 @@
+server {
+  listen 80;
+  listen 443;
+  listen 3002;
+
+  server_name mock-server;
+  root /www/test-data/;
+
```

Comment:
We should set the `Content-Type` to `application/json`. This makes it easy to hit the mock api in the browser or postman.